### ISSUE TITLE
Internal issue: b/31320843

### ISSUE THREAD

**Issue by jverkoey** [material-components/material-components-ios#4130]
This is an internal issue. If you are a Googler, please visit [b/31320843](http://b/31320843) for more details.

---
#### Internal data
- Associated internal bug: [b/31320843](http://b/31320843)

**Comment by ianegordon** [comment]
This is blocked on an MDC implementation of Speeddial.

**Comment by romoore** [comment]
Closing as no longer relevant to MDC and is now an Google-internal issue.
